embark-consult.el

```diff
@@ -66,6 +66,14 @@
 (eval-when-compile
   (require 'cl-lib))
 
+(defcustom embark-consult-export-location-candidates-to-grep-mode nil
+  "If set to true, `grep-mode' is used to export results of
+`consult-line', `consult-line-multi' and other `consult-location'
+enabled commands.
+
+If set to nil `occur-mode' is used."
+  :type 'boolean )
```

Comment:
This variable is not necessary. Users can simply adjust the exporter alist directly.

~~~ elisp
(setf (alist-get 'consult-location embark-exporters-alist) ...)
~~~